src/types.ts

```diff
@@ -108,3 +122,14 @@ export enum GitHubOrderBy {
   Followers = 'followers',
   Recent = 'recent',
 }
+
+// Combine Internship + New Grad orderings
```

Comment:
is this comment needed? might at least reword if you leave it in

Reply:
Removed